templates/org/aac/aac-architecture-guide-content.md

```diff
@@ -1,66 +1,70 @@
-description: <Write a 100-160 character description that ends with a period and ideally starts with a call to action. This becomes the browse card description.>
-author: <Contributor's GitHub username>
+title: <Article Title in Title Casing>
+description: <Align with the template instructions for this content type.>
+author: <Contributor's GitHub user name>
 ms.author: <Contributor's Microsoft alias>
 ms.date: <Date of publish or last freshness pass, in mm/dd/yyyy format>
 ms.topic: conceptual
 ms.service: azure-architecture-center
 ms.subservice: architecture-guide
 products:
-  - <Choose 1-5 products from the list at https://review.learn.microsoft.com/help/contribute/architecture-center/aac-browser-authoring#products>
+  - <Choose 1-5 products, as explained in the template instructions.>
   - <1-5 products>
   - <1-5 products>
 categories:
-  - <Choose at least one category from the list at https://review.learn.microsoft.com/help/contribute/architecture-center/aac-browser-authoring#azure-categories>
-  - <There can be more than one category>
+  - <Choose at least one category, as explained in the template instructions.>
+  - <There can be more than one category.>
 ---
 
-# H1 title
+# <!-- Article title in sentence casing -->
 
-Other than the H2s listed below, there are no further content requirements for Architecture Guides, except to follow all Microsoft and Learn style and pull request criteria. Diagrams are not required for this content type, but if a diagram is included, it should have a Dataflow/Workflow section underneath the diagram to explain it. See the [Example workload template](sample-solution-templates) sections on Architecture and Dataflow for details.
+<!-- Add a brief introductory paragraph. -->
 
-The Architecture Guide template requires the following sections at the end of the article:
-  
-## Contributors
+## Architecture
 
-> (Expected, but this section is optional if all the contributors would prefer to not include it)
+<!-- This section is optional. Use the following format to link to your image file:-->
 
-> Start with the explanation text (same for every article), in italics. This makes it clear that Microsoft takes responsibility for the article (not the one contributor). Then include the "Pricipal authors" list and the "Additional contributors" list (if there are additional contributors) (all in plain text, not italics or bold). Link each contributor's name to the person's LinkedIn profile. After the name, place a pipe symbol ("|") with spaces, and then enter the person's title. We don't include the person's company, MVP status, or links to additional profiles (to minimize edits/updates). (The profiles can be linked to from the person's LinkedIn page, and we hope to automate that on the platform in the future). 
-> Implement this format:
+:::image type="content" border="false" source="./images/<file-name>.png" alt-text="Diagram that shows the <solution name> architecture." lightbox="./images/<file-name>.png":::
```

Comment:
Would it be okay if we used the "long description" format for this link by default?  Our images are fairly complex, and Microsoft Learn calls out that [complex images SHOULD come in the format of long description](https://review.learn.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main#complex-images).

```suggestion
:::image type="complex" source="./images/<file-name>.png" alt-text="Diagram that shows the <solution name> architecture." lightbox="./images/<file-name>.png":::
    <long description here>
:::image-end:::
```